tools/legacy/BuildMonitor/Legacy_BuildMonitor.csproj

```diff
@@ -1,225 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
This seems like a lot to delete without having a replacement. Are we sure this can be tossed so easily?

I understand it's all "legacy" but I'd simply like to be sure.

Reply:
1. All the code will be available for eternity in git history
2. This doesn't compile because VS2017 stopped shipping the VS2015 extensibility toolkit.
3. VS2017 and new rule sets largely obviated the need for these guys.